app/views/mentoring/external/show.html.haml

```diff
@@ -5,9 +5,11 @@
       %h1 Share your knowledge with others
       %p
         Mentoring is a fun and rewarding way to reinforce your own learning, while helping others learn and discover things they don’t know.
+        - # TODO
```

Comment:
@ErikSchierboom Link here for `using/concepts/reputation`. Maybe put a few sentences on what reputation is (user-facing) and a table that shows what rep you get for what? But whatever's quick.

(If you can find a better word than concepts too, that'd be good!)

Reply:
I'm gonna refer them to https://exercism.org/docs/building/product/reputation for now, okay?